data/json/items/tool/woodworking.json

```
@@ -249,7 +249,7 @@
   {
     "id": "primitive_axe",
     "type": "GENERIC",
-    "category": "tools",
+    "category": "tools_workshop",
```

Other axes are also in farming tools
```suggestion
    "category": "tools_farming",
```